mathics/docpipeline.py

```diff
@@ -257,6 +255,10 @@ def show_test_summary(
     return
 
 
+#
+#  TODO: Split and simplify this section
+#
+#
```

Comment:
I guess this function still requires work. In general, `test_tests/test_sections/test_chapters/test_all` doesn't seem to take advantage of the new organization in ``common_doc``.